Quadriga/src/main/java/edu/asu/spring/quadriga/service/conceptcollection/IConceptCollectionManager.java

```diff
@@ -127,5 +127,8 @@ void addItems(String lemmma, String id, String pos, String desc,
 
 
 	public abstract String getProjectsTree(String userName,String ccId) throws JSONException;
+
+
+    String getConceptDescriptionFromConceptId(String id);

```

Comment:
Please format this class so that all method declaration start at the same position. Thanks.